packages/blockchain-info-components/index.d.ts

```diff
@@ -175,6 +176,7 @@ export const Link: FunctionComponent<{
   onClick?: (e?: KeyboardEvent) => void
 }>
 export const Modal: FunctionComponent<{
```

Comment:
minor it could be pure functional component no need to use react `FunctionComponent` - but it's not breaker or stopper ;)